### PR TITLE
Fix zero-width emotes sometimes wrapping lines incorrectly

### LINKED ISSUE #3147: Zero-width emotes can't be last on line when text wrapped
**Describe your issue**
Zero-width emotes can't be last on line when text wrapped. See screenshot

**Screenshots**
Screenshot at 2021-08-08 01-08-53
Right most row of pepeL emotes is not covered in cvHazmat. The cvHazmat emotes instead are wrapped to new line

**OS and Chatterino Version**
OS: `Linux mainPC 5.10.0-8parrot1-amd64 #1 SMP Debian 5.10.46-8parrot1 (2021-07-21) x86_64 GNU/Linux`
Chatterino: `Chatterino 7.3.3 (commit 20a7127)`

### PR DESCRIPTION
**Pull request by acdvs**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description

Prevents trailing zero-width emotes from individually line breaking by considering their width as 0 instead of the calculated width of a space. For example, typing `LUL cvHazmat` will result in only the width of `LUL`, with `cvHazmat` completely disregarded. This in turn also prevents zero-width emotes from being shifted out of bounds to the left when individually line broken.

One side effect of this change is that stacked zero-width emotes (e.g., `cvHazmat cvHazmat cvHazmat`) will run out of bounds to the right if at the end of a line since they would now only be considered overflow from the base emote (e.g., `LUL`).

Fixes #3147

### CODE REVIEW

**Review by Felanbird — COMMENTED** [review]
This PR works exactly as intended - just needs a formatting fix. 
Also I built this change against chatterino7 just to check the 7TV emotes, and those are fixed as well.

I might suggest a more user friendly changelog wording but that can be done by a maintainer before merge.

**Review by Mm2PL — APPROVED** [review]
Needs a better changelog entry and reformat, but that can be done by the maintainer who's gonna merge this.

File: CHANGELOG.md
Comment (Felanbird) [inline CHANGELOG.md:6]:
```suggestion
- Bugfix: Fixed zero-width emotes sometimes wrapping lines incorrectly. (#3389)
```
Will have to drop down to the bugfix section

**Review by pajlada — APPROVED** [review]
Awesome! Thanks for fixing this!

As a first-time contributor, you can now add yourself to the contributors list that's shown inside the application.

To do so, open a new PR where you modify the `resources/contributors.txt` file and add yourself to the list. Make sure to read the comments at a top for instructions.

**Review by Felanbird — APPROVED** [review]

**Review by zneix — APPROVED** [review]

**Pull request merged**
